requirements.txt

```diff
@@ -20,7 +20,7 @@ git+git://github.com/sensu/sensu-plugin-python@bf00ac33
 # so we pull from git
 git+git://github.com/thefactory/marathon-python@c1e0c341#egg=marathon
 # While task_processing is in rapid development mode, we install it from git
-git+git://github.com/Yelp/task_processing@95cc2f8b#egg=task_processing[mesos_executor]
+git+git://github.com/Yelp/task_processing@29e4361e#egg=task_processing[mesos_executor]
```

Comment:
The same reason *anything* is pinned in requirements.txt -- so your build isn't broken all the time (reproducibility) -- this PR is a great example of *exactly why* this is a problem (the modified test would have been broken and all the paasta builds would have been broken -- likely taking **hours** of someone's time as they waited for travis-ci to rerun).